### ISSUE TITLE
Create a media library selection form component

### ISSUE THREAD

**Issue by JanLenoch** [Kentico/xperience-training-13#68]
### Who

- [ ] Anonymous visitor
- [ ] Patient
- [ ] Doctor
- [x] Content editor
- [ ] Marketer
- [ ] Administrator
- [ ] Developer

### Needs what

To select a media library for the media library uploader form component.

### Why

Why is the feature needed?

### Issue expands upon

This issue expands upon another issue(s): #67 

### Blockers

This issue is blocked by the following issue(s): #`issue number`

**Comment by JanLenoch** [comment]
Resolved via e0cb981e0b0e88c76070572e7da089d81b807c78
